# GRAD521_DMPTopham_2023

Data Management Plan for the research project The role of newly identified luteovirus in recurrence of blueberry shock virus

Context of the project:

The four main goals of the project are as follows:
	Evaluate advanced breeding selections, historical cultivars, and wild Vaccinium species for virus symptoms and for presence of blueberry shock virus and luteovirus.
	Evaluate historical cultivars and wild vaccinium species for signs of tolerance or resistance to shock for further investigation.
	Investigate symptomatic plant genotypes that tested negative for blueberry shock virus, luteovirus, or both and subject those to high throughput sequencing (HTS). 
	Analyze HTS data for new shock and luteovirus isolates (genomic variants) as well as for other viruses including new and emerging viruses. 

The types of data:
1.	Physical samples: Approximately 2,000 blueberry leaf tissue samples from Pacific Northwest advanced breeding selections, historical cultivars, and wild Vaccinium species per collection year are to be collected over an anticipated 2-3 year study. A subsample from each sample will be lyophilized and stored at -20 C in preparation of laboratory analysis with the bulk of the sample stored at -80 C. An inventory of the samples collected will be maintained via a template created on a spreadsheet software. (less than 1 MB)
2.	Field observations: There will be hard-copy data sheets containing observations made in the field mapping the sample plots, matching samples to their individual ids, and indicated any viral symptoms observed. These will be subsequently transcribed for use in compiling datasets. The data denoting the specific growers collected from will need to be protected and not disclosed. (less than 1 MB)
3.	Laboratory data: These consist of hard-copy procedural documentation and digital files that contain information recorded during lab processing. This includes nucleic extraction parameters, RT-PCR parameters and results in the form of electrophoresis imaging, and Rapid Amplification of cDNA Ends (RACE) processing. Transcribed and instrument outputs are kept in raw form and referenced for the creation of compiled data sets. (10 MB)
4.	Compiling datasets: The generated dataset is to merge sample descriptors, lab processing and output data, and computation of statistical trends across the entire dataset and subsets of interest to prepare it for analysis. (less than 1 MB)
5.	Computational data: The data generated in the lab as well as from existing databases (NCGI) and collaborators (high throughput sequencing results), will need to be processed via computational software in order to annotate open reading frames (ORFs), align the novel sequence to known viral sequences, and complete phylogenetic analysis on the aligned sequences. Additional statistical analysis will be performed on the viral testing results. (10 MB)
6.	Historical records: This includes data type 4 listed above. They exist in the in a digital excel spreadsheet. (less than 1 MB)  
